training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/submit_smp_trainer.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        hyperparameters["fp16"] = True
</pre><p>Why 2 fp16 settings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3577/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Oh yeah, I'll change it to one that sets both. Originally was to make where they were respectively set look more uniform.</p>